anydex/test/wallet/monero/test_xmr_wallet.py

```diff
@@ -540,6 +542,30 @@ async def test_get_confirmations_wallet(self):
         self.assertEqual(4, await w.get_confirmations(p))
         w.cancel_all_pending_tasks()
 
+    def test_new_address(self):
```

Comment:
Is it okay if I copy all of the new code in xmr wallet and xmr tests to their correct places in the refactor branch? Just so that we don't forget what is new and how to deal with the collisions by the time we start merging everything together.